packages/data-stores/src/site/test/resolvers.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * These tests shouldn't require the jsdom environment,
+ * but we're waiting for this PR to merge:
+ * https://github.com/WordPress/gutenberg/pull/20486
```

Comment:
Same as https://github.com/Automattic/wp-calypso/pull/40158/files#r398592801

Reply:
Looks like we might still need it [for the time being](https://github.com/Automattic/wp-calypso/pull/40158#discussion_r399072574)